atlasdb-container-test-utils/src/main/java/com/palantir/atlasdb/containers/ThreeNodeCassandraClusterOperations.java

```diff
@@ -87,11 +93,29 @@ private String runCql(String cql) throws IOException, InterruptedException {
 
     private String runNodetoolCommand(String nodetoolCommand, int timeoutSeconds) throws IOException,
             InterruptedException {
+        return runNodetoolCommandOnContainer(nodetoolCommand, timeoutSeconds,
+                ThreeNodeCassandraCluster.FIRST_CASSANDRA_CONTAINER_NAME);
+    }
+
+    private List<String> runNodetoolCommandOnAllContainers(String nodetoolCommand, int timeoutSeconds)
+            throws IOException, InterruptedException {
+        List<String> ret = Lists.newArrayList();
+        ret.add(runNodetoolCommandOnContainer(nodetoolCommand, timeoutSeconds,
```

Comment:
Use a stream/loop here after adding a constant for `List<CassandraContainers>`.

Reply:
again +1; an ImmutableList would also work.

Reply:
Done. Stream didn't work, as `runNodetoolCommandOnContainer` throws checked exceptions.